app/views/import_users/import.html.erb

```diff
@@ -1,9 +1,12 @@
 <% title "Import Users" %>
 
 <h3>CSV Import Format</h3>
+  <p>
+    To import user data, upload a CSV file that follows this format. All fields except "nickname" are required.
+  </p>
 <h5>Columns (must be in the first row of the CSV file)</h5>
 <%= markdown("
-    Username, First Name, Last Name, Nickname, Phone, Email, Affiliation") %>
```

Comment:
actually, this page is now more similar to the equipment import page! (which is entirely incidental but still!)